plugin/execute_command.py

```diff
@@ -15,6 +15,10 @@ def run(self,
             command_name: Optional[str] = None,
             command_args: Optional[List[Any]] = None,
             event: Optional[dict] = None) -> None:
+        # Handle VSCode-specific command for triggering suggestions popup.
+        if command_name == "editor.action.triggerSuggest":
+            sublime.set_timeout(lambda: self.view.run_command("auto_complete"))
+            return
```

Comment:
Why does this need a set_timeout? I think it can be `return self.view.run_command("auto_complete")` no?

Reply:
Just doesn't work without it. Possibly something to do with running one command within another command (and possibly specific to auto_complete).

Reply:
Oh okay. The magical set_timeout solves most of the mysterious problems ;)